frame/nfts/src/types.rs

```diff
@@ -40,6 +43,12 @@ pub(super) type ItemTipOf<T, I = ()> = ItemTip<
 	BalanceOf<T, I>,
 >;
 
+pub trait Incrementable {
+	fn increment(&self) -> Self;
+	fn initial_value() -> Self;
+}
+impl_incrementable!(u8, u16, u32, u64, u128, i8, i16, i32, i64, i128);
```

Comment:
You can probably implement this for all `Saturating + Zero` types.

Reply:
same as above